app/models/migration/ecf/finance/schedule.rb

```diff
@@ -0,0 +1,10 @@
+module Migration::Ecf::Finance
+  class Schedule < Migration::Ecf::BaseRecord
+    self.inheritance_column = nil
+
+    belongs_to :cohort
+    has_many :participant_profiles
+
+    default_scope { where("schedules.type ilike ?", "Finance::Schedule::NPQ%") }
```

Comment:
```suggestion
    default_scope { where(type: "Finance::Schedule::NPQ") }
```

Reply:
Hi @cwrw Think we need to keep the `ilike ?", "Finance::Schedule::NPQ%"` as the schedules types in ECF are named as for example: `"Finance::Schedule::NPQEhco",
 "Finance::Schedule::NPQSpecialist",
 "Finance::Schedule::NPQLeadership",
 "Finance::Schedule::NPQSupport", etc`

Reply:
ah right gotcha! 👍